starknet_devnet/server.py

```diff
@@ -43,7 +47,7 @@ async def add_transaction():
     else:
         abort(Response(f"Invalid tx_type: {tx_type}.", 400))
 
-    await starknet_wrapper.postman_flush()
+    dumper.dump_if_required()
```

Comment:
@dribeiro-ShardLabs I accidentally removed the flush line, I will return it so it's not part of this PR, but just wondering - why did the postman tests pass anyway?

Reply:
Received response in a separate channel: because of the flush in call endpoint.